arches_lingo/src/arches_lingo/utils.ts

```diff
@@ -53,6 +59,66 @@ export function navigateToSchemeOrConcept(
     }
 }
 
+/* Port of rank_label in arches.app.utils.i18n python module */
+export const rankLabel = (
```

Comment:
What do you think about just importing these from controlled lists?

Reply:
yep! sg2m, I'll roll that into the next iteration of this PR